404.html

```diff
@@ -46,30 +68,54 @@
         <div class="wrap">
             <h1 class="title"><span id="nametext">Anonymous,</span></h1>
             <h1 class="title">This is a full stop, not a period, full stop.</h1>
-            <a href="pages/textAnimation.html">Really?</a>
+            <h2 id="curious" onClick="dictionaryDefinition()">What?</a>
+            <h2 id="dictionary-definition">Full Stop (/ˈfʊl ˈstɑp/) - What you Americans call a period. </span>
+            <br>
+            <a id="animations" style="margin-top: 20px;" href="pages/textAnimation.html">Really?</a>
         </div>
     </div>
 </body>
 <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.6.4/jquery.min.js"></script>
 
 <script type="text/javascript">
-    $(document).ready(function () {
+        
+        
+    //defining on click function for What?    
+    function dictionaryDefinition(){
+    	$("#curious").one('click', function(e) {
+    		$('#dictionary-definition').css("display", "block");
+    		$('#animations').css("display", "block");
+    	});
+    }
 
+    $(document).ready(function () {
+		
+		//hide dictionary-definition and Really?
```

Comment:
This may fix your issue, as you don't need to use the (document).ready function.